app/js/directives/yesNoQuestion.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('ddsApp').directive('yesNoQuestion', function() {
+    return {
+        restrict: 'E',
+        transclude: true,
+        templateUrl: 'partials/foyer/yes-no-question.html',
+        scope: true,
+        controller: 'yesNoQuestionCtrl',
+        link: function ($scope, $element, $attributes) {
+            $scope.modelString = $attributes.model;
+        }
+    };
+});
+
+angular.module('ddsApp').controller('yesNoQuestionCtrl', function($scope, $transclude, $element, $interpolate) {
+	_.forEach($transclude(), function(elem) {
```

Comment:
I'm kind of reimplementing pieces of angular here, but multi-slot transcluding is not implemented until 1.5, and without it we need to pass the texts in attributes, which is less convenient and have limitations.